Azure/sonic-buildimage
# dhcp6relay: remove line overwriting docker-dhcp-relay variable

The dhcp6relay rules file had a line overwriting a variable for
docker-dhcp-relay. Remove that line.

This line caused a limited impact where if some (many?) of the docker
containers were already built, except for dhcp-relay, and the build
failed or was interrupted, then dhcp-relay container would fail to build
because this variable was overwritten and the python3-swsscommon
wouldn't get installed into the slave container. Most builds would be
fine, though.

Signed-off-by: Saikrishna Arcot <sarcot@microsoft.com>

#### Why I did it

#### How I did it

#### How to verify it

To see the failure:
1. Start a regular build (for any platform).
2. When the containers are getting built, before the dhcp-relay container is built, interrupt the build with Ctrl-C.
3. If it's timed right, after restarting the build, the dhcp-relay container fails to build.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
